src/VolumeDrawable.ts

```diff
@@ -324,44 +318,64 @@ export default class VolumeDrawable {
         this.setVolumeRendering(false);
       }
     }
+    if (this.settings.viewAxis !== axis) {
+      this.settings.viewAxis = axis;
+      this.volumeRendering.updateSettings(this.settings, SettingsFlags.VIEW);
+    }
   }
 
   // Tell this image that it needs to be drawn in an orthographic mode
   // @param {boolean} isOrtho is this an orthographic projection or a perspective view
   setIsOrtho(isOrtho: boolean): void {
-    this.volumeRendering.setIsOrtho(isOrtho);
+    if (this.settings.isOrtho === isOrtho) {
+      return;
+    }
+    this.settings.isOrtho = isOrtho;
+    this.volumeRendering.updateSettings(this.settings, SettingsFlags.VIEW);
   }
 
   setInterpolationEnabled(active: boolean): void {
-    this.volumeRendering.setInterpolationEnabled(active);
+    if (this.settings.useInterpolation === active) {
+      return;
+    }
+    this.settings.useInterpolation = active;
+    this.volumeRendering.updateSettings(this.settings, SettingsFlags.SAMPLING);
   }
 
   setOrthoThickness(value: number): void {
     !this.PT && this.meshVolume.setOrthoThickness(value);
-    this.volumeRendering.setOrthoThickness(value);
+    // No settings update because ortho thickness is calculated in the renderers
   }
 
   // Set parameters for gamma curve for volume rendering.
   // @param {number} gmin 0..1
   // @param {number} glevel 0..1
   // @param {number} gmax 0..1, should be > gmin
   setGamma(gmin: number, glevel: number, gmax: number): void {
-    this.gammaMin = gmin;
-    this.gammaLevel = glevel;
-    this.gammaMax = gmax;
-    this.volumeRendering.setGamma(gmin, glevel, gmax);
+    if (this.settings.gammaMin === gmin || this.settings.gammaLevel === glevel || this.settings.gammaMax === gmax) {
+      return;
+    }
+    this.settings.gammaMin = gmin;
+    this.settings.gammaLevel = glevel;
+    this.settings.gammaMax = gmax;
+    this.volumeRendering.updateSettings(this.settings, SettingsFlags.CAMERA);
   }
 
   setFlipAxes(flipX: number, flipY: number, flipZ: number): void {
-    this.flipX = flipX;
-    this.flipY = flipY;
-    this.flipZ = flipZ;
-    this.volumeRendering.setFlipAxes(flipX, flipY, flipZ);
-    this.meshVolume.setFlipAxes(flipX, flipY, flipZ);
+    const flipAxes = new Vector3(flipX, flipY, flipZ);
```

Comment:
I wonder if the type hints for the args here could be `-1 | 1` instead of `number`